src/commercetools/testing/abstract.py

```diff
@@ -118,6 +118,16 @@ def delete_by_key(self, key):
             if obj["key"] == key:
                 return self.objects.pop(obj_id)
 
+    def delete_by_container_key(self, container, key):
+        return next(
+            (
+                obj
+                for obj in self.objects.values()
+                if obj["container"] == container and obj["key"] == key
+            ),
+            None,
+        )
```

Comment:
Why not just?
```suggestion
    def delete_by_container_key(self, container, key):
         for obj_id, obj in self.objects.items():
             if obj["key"] == key and obj["container"] == container:
                 return self.objects.pop(obj_id)
        )
```

To have it in line with `delete_by_key` above?